Gemfile.lock

```diff
@@ -55,13 +55,6 @@ GEM
       xpath (>= 2.0, < 4.0)
     childprocess (0.8.0)
       ffi (~> 1.0, >= 1.0.11)
```

Comment:
I think this file should not be here. If you change something in Gemfile, you should commit these changes along with Gemfile.lock. Commiting only one of this files is not a good practice. Consider reverting this changes or add Gemfile to this PR as well